tools/ci/build_autoinst.sh

```diff
@@ -8,6 +8,7 @@ set -ex
 destdir=${1:-../os-autoinst}
 sha=${2:-$(cat tools/ci/autoinst.sha)}
 
+sudo chown -R $USER $destdir/..
```

Comment:
when you understood why we need this would be great to have a comment on top explaining the reason :)

Reply:
I'm doing this because I get permission errors. The folder outside the project itself is not owned by the user.

Reply:
hm, ok. Well, leave as is then. But now there is https://github.com/os-autoinst/openQA/runs/4284016039?check_suite_focus=true#step:5:9